multipoint/multiPoint.py

```diff
@@ -90,7 +90,7 @@ def addProcessorSet(self, setName, nMembers, memberSizes):
             memberSizes = numpy.ones(nMembers) * memberSizes[0]
         else:
             if len(memberSizes) != nMembers:
-                print("Error: The suppliled memberSizes list is not the correct length")
+                print("Error: The suppllied memberSizes list is not the correct length")
```

Comment:
typo here

Reply:
I think you're reading an older commit or something, this file has been removed. Could you double check the rest to make sure your review didn't miss any code changes?

Reply:
You are right, I added this comment following the commit history but I did not check the final state of the code - and missed the `outdated` label when checking by review. Everything else is fine.